code/game/machinery/kitchen/microwave.dm

```diff
@@ -16,7 +16,7 @@
 	var/global/list/datum/recipe/available_recipes // List of the recipes you can use
 	var/global/list/acceptable_items // List of the items you can put in
 	var/global/list/acceptable_reagents // List of the reagents you can put in
-	var/global/max_n_of_items = 0
+	var/global/max_n_of_items = 20

```

Comment:
Previously max number of items was set to the maximum quantity that any single recipe used. Since the microwave is now for bulk cooking, that is no longer applicable. Set it to a sizeable flat capacity instead